services/workflows-service/src/workflow/workflow-collection-flow.controller.ts

```diff
@@ -0,0 +1,77 @@
+import { CollectionFlowStateService } from '@/collection-flow/collection-flow-state.service';
+import { CollectionFlowMissingException } from '@/collection-flow/exceptions/collection-flow-missing.exception';
+import { AppLoggerService } from '@/common/app-logger/app-logger.service';
+import { CurrentProject } from '@/common/decorators/current-project.decorator';
+import type { TProjectId } from '@/types';
+import * as common from '@nestjs/common';
+import * as swagger from '@nestjs/swagger';
+import { UpdateCollectionFlowStateDto } from './dtos/update-collection-flow-state.dto';
+import { CollectionFlowStateSchema } from '@ballerine/common';
+import { Type } from '@sinclair/typebox';
+
+export const WORKFLOW_TAG = 'WorkflowsCollectionFlow';
+@swagger.ApiBearerAuth()
+@swagger.ApiTags(WORKFLOW_TAG)
+@common.Controller('external/workflows/collection-flow')
```

Comment:
You reckon this controller shouldn't be merged with one of the collection flow controllers or the workflows controllers?